src/aig-common/modules/standard/components/ipp-lot-category/lot-category-form/lot-category-form.component.html

```diff
@@ -0,0 +1,38 @@
+<form *ngIf="step.form" [formGroup]="categoryNewForm" (ngSubmit)="createCategory()">
```

Comment:
name not correct:
`[formGroup]="categoryNewForm"`
correct name
`[formGroup]="lotCategoryNewForm"`